### ISSUE TITLE
When running tests, can't find variable: QUnit

### ISSUE THREAD

**Issue by asakusuma** [emberjs/ember.js#13331]
Not sure if this issue belongs here, or in testem, but I've only seen this issue in the ember.js repo, not local addons.

Running `npm run test` or any testem command results in a error about QUnit not being found. This is on a brand new clone.

```
// testem
Uncaught ReferenceError: QUnit is not defined at http://localhost:7357/5163/tests/index.html?hidepassed, line 23
Uncaught SyntaxError: Unexpected token { at http://localhost:7357/5163/tests/index.html?hidepassed, line 67
Uncaught ReferenceError: QUnit is not defined at http://localhost:7357/5163/tests/index.html?hidepassed, line 80
Uncaught ReferenceError: QUnit is not defined at http://localhost:7357/5163/tests/index.html?hidepassed, line 94
Uncaught ReferenceError: QUnit is not defined at http://localhost:7357/5163/tests/index.html?hidepassed, line 105
Uncaught ReferenceError: QUnit is not defined at http://localhost:7357/5163/tests/index.html?hidepassed, line 180
Uncaught ReferenceError: QUnit is not defined at http://localhost:7357/5163/tests/index.html?hidepassed, line 198
```

```
// npm test
ReferenceError: Can't find variable: QUnit


  undefined:5
  :0 in exit
  phantomjs://code/runner.js:56 in onCallback
  phantomjs://platform/webpage.js:146 in connector
  :0 in call
  :1 in callPhantom
  :55
  http://localhost:13141/qunit/qunit.js:313 in runLoggingCallbacks
  http://localhost:13141/qunit/qunit.js:699 in done
  http://localhost:13141/qunit/qunit.js:637 in process
  http://localhost:13141/qunit/qunit.js:611 in begin
  http://localhost:13141/qunit/qunit.js:671
```

### index.html issues
The index.html test harness has 2 issues.

First, the QUnit path is wrong. It's attempting to load `http://localhost:7357/8160/qunit/qunit.js`, when the qunit file is really at `http://localhost:7357/8160/dist/qunit/qunit.js`

Second, this line exists in index.html, which looks like some template that was not actually compiled.
`window.ENV.FEATURES = !!QUnit.urlParams.prod ? {{PROD_FEATURES}} : {{DEV_FEATURES}};`

npm version 2.14.2
node version 4.0.0
ember-cli: 2.4.3
global testem: 1.6.0

cc @stefanpenner

**Comment by pixelhandler** [comment]
@asakusuma what are the steps to reproduce this issue? Do I clone emberjs/ember.js and run tests; e.g. following the contributing guide for this repo? https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md#building-emberjs

**Comment by asakusuma** [comment]
I followed those steps exactly and then ran npm run test. I'm guessing there is a build issue caused by something local to my machine interfering. The un-compiled placeholders and the wrong qunit path are the two biggest clues and I was hoping someone would have some idea based on those clues.

Running the tests in the browser via ember serve works. Running npm run test does not work, and neither does any direct testem command.

The reason this is blocking me is because I need to manually kick off a saucelab job directly from my computer to debug the failing Edge saucelab test run here: https://github.com/emberjs/ember.js/pull/13328

**Comment by asakusuma** [comment]
Pretty sure the issue is that the `/tests/index.html` is being executed, instead of `/dist/tests/index.html`. This would explain both the path being wrong and the uncompiled placeholders.

**Comment by rwjblue** [comment]
We generally do not run `testem` directly, we use `ember test --server` to do that. Which sets up the dist paths slightly differently.

**Comment by asakusuma** [comment]
@rwjblue makes sense. However, I am trying to kick off a saucelabs test run from my machine.

To workaround the issue locally, I changed [this line](https://github.com/emberjs/ember.js/blob/master/testem.json#L3) to `dist/tests/index.html?hidepassed`, i.e. just manually adding the `/dist` part of the path.

**Comment by rwjblue** [comment]
When running `testem` from the command line, you may need to use `testem.dist.json` instead which does use `dist/tests/index.html` instead of `tests/index.html`: https://github.com/emberjs/ember.js/blob/master/testem.dist.json#L3

**Comment by asakusuma** [comment]
I was able to fix this by downgrading to phantomjs `1.9.8`, which is the version being run by travis.
